lib/yelp/error.rb

```diff
@@ -83,7 +83,8 @@ def initialize(msg='One or more parameters were invalid', error=nil)
         unless error.nil?
           @text = error['text']
           @field = error['field']
-          msg = msg + ': ' + @field
+          description = error.has_key?('description') ? '. Description: ' + error['description'] : ''
+          msg = msg + ': ' + @field + description
```

Comment:
I like your suggestion. Reasoning here for not using the double quotes interpolation is the same as with the testing style. I was trying to follow what seemed to be a convention stablished before.